content/lab01/cells-organelles-polarity-amp-cellular-specializations.md

```diff
@@ -1,33 +1,33 @@
 ---
-title: "Cells, Organelles: Polarity &amp; Cellular Specializations"
```

Comment:
Pointing out the switch from entity reference to ampersand. Is that intended?

Reply:
@sdreher Yes, that's intended. Hugo renders it as escaped HTML.  I caught that one but I should grep for other occurrences.

Reply:
+1